ca/certificate-authority.go

```diff
@@ -156,7 +120,11 @@ func NewCertificateAuthorityImpl(cadb core.CertificateAuthorityDatabase, config
 		return nil, err
 	}
 
-	pa := policy.NewPolicyAuthorityImpl()
+	pa, err := policy.NewPolicyAuthorityImpl(commonConfig)
```

Comment:
Well, know better what it requires.

I'm pretty nervy about all the disk IO that is happening in this function already. I think we need to pull all of this key reading back out.